test/cpp/jit/test_gpu.cpp

```diff
@@ -18116,6 +18117,76 @@ TEST(NVFuserTest, FusionIssue1133_CUDA) {
   testValidate(&fusion, outputs, aten_inputs, {ref}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, FusionDefaultGemmSchedule_CUDA) {
```

Comment:
I think I mentioned this somewhere else, but let's use a new test file for MMA tests, e.g., `test_gpu_mma.cpp`.